README.md

```diff
@@ -7,10 +7,10 @@ delete, and enumerate
 resources to be managed by Chef.
 
 ## Installation
-Be sure you are running the latest version of Chef, which can be installed
+Be sure you are running the latest version of Chef DK, which can be installed
 via:
 
-    gem install chef
+    https://downloads.chef.io/chef-dk/
 
 This plugin is distributed as a Ruby Gem. To install it, run:
```

Comment:
I think If you are using chef-dk in that case ruby gems are install using -
`chef gem install <name>`. And also chef-dk comes with inbuilt knife azure, we have to verify how `chef gem install knife-azure` will work here, As recently we found some gem dependency related issues with chef-dk.

Thank you for sharing this.